core/src/main/java/org/infinispan/reactive/publisher/impl/LocalPublisherManagerImpl.java

```diff
@@ -34,7 +34,6 @@
 import org.infinispan.factories.impl.ComponentRef;
 import org.infinispan.factories.scopes.Scope;
 import org.infinispan.factories.scopes.Scopes;
-import org.infinispan.reactive.RxJavaInterop;
```

Comment:
This import is still required. Compilation is failing because this was removed:

```java
[2023-12-11T06:00:50.069Z] 01:00:48.737 [INFO] -------------------------------------------------------------
[2023-12-11T06:00:50.069Z] 01:00:48.737 [ERROR] COMPILATION ERROR : 
[2023-12-11T06:00:50.069Z] 01:00:48.737 [INFO] -------------------------------------------------------------
[2023-12-11T06:00:50.069Z] 01:00:48.737 [ERROR] /home/infinispan/workspace/Infinispan_PR-11560/core/src/main/java/org/infinispan/reactive/publisher/impl/LocalPublisherManagerImpl.java:[189,17] cannot find symbol
[2023-12-11T06:00:50.069Z]   symbol:   variable RxJavaInterop
[2023-12-11T06:00:50.069Z]   location: class org.infinispan.reactive.publisher.impl.LocalPublisherManagerImpl<K,V>
[2023-12-11T06:00:50.069Z] 01:00:48.737 [ERROR] /home/infinispan/workspace/Infinispan_PR-11560/core/src/main/java/org/infinispan/reactive/publisher/impl/LocalPublisherManagerImpl.java:[190,16] incompatible types: io.reactivex.Flowable<java.lang.Object> cannot be converted to io.reactivex.Flowable<org.infinispan.container.entries.CacheEntry<K,V>>
[2023-12-11T06:00:50.069Z] 01:00:48.737 [ERROR] /home/infinispan/workspace/Infinispan_PR-11560/core/src/main/java/org/infinispan/reactive/publisher/impl/LocalPublisherManagerImpl.java:[556,16] cannot find symbol
[2023-12-11T06:00:50.069Z]   symbol:   variable RxJavaInterop
[2023-12-11T06:00:50.069Z]   location: class org.infinispan.reactive.publisher.impl.LocalPublisherManagerImpl<K,V>
[2023-12-11T06:00:50.069Z] 01:00:48.737 [ERROR] /home/infinispan/workspace/Infinispan_PR-11560/core/src/main/java/org/infinispan/reactive/publisher/impl/LocalPublisherManagerImpl.java:[577,29] cannot find symbol
[2023-12-11T06:00:50.069Z]   symbol:   variable RxJavaInterop
[2023-12-11T06:00:50.069Z]   location: class org.infinispan.reactive.publisher.impl.LocalPublisherManagerImpl<K,V>
[2023-12-11T06:00:50.069Z] 01:00:48.737 [ERROR] /home/infinispan/workspace/Infinispan_PR-11560/core/src/main/java/org/infinispan/reactive/publisher/impl/LocalPublisherManagerImpl.java:[567,35] incompatible types: inference variable I has incompatible equality constraints org.infinispan.container.entries.CacheEntry<K,V>,java.lang.Object
[2023-12-11T06:00:50.069Z] 01:00:48.737 [ERROR] /home/infinispan/workspace/Infinispan_PR-11560/core/src/main/java/org/infinispan/reactive/publisher/impl/LocalPublisherManagerImpl.java:[598,26] cannot find symbol
[2023-12-11T06:00:50.069Z]   symbol:   variable RxJavaInterop
[2023-12-11T06:00:50.069Z]   location: class org.infinispan.reactive.publisher.impl.LocalPublisherManagerImpl<K,V>
[2023-12-11T06:00:50.069Z] 01:00:48.737 [ERROR] /home/infinispan/workspace/Infinispan_PR-11560/core/src/main/java/org/infinispan/reactive/publisher/impl/LocalPublisherManagerImpl.java:[684,17] cannot find symbol
[2023-12-11T06:00:50.069Z]   symbol:   variable RxJavaInterop
[2023-12-11T06:00:50.069Z]   location: class org.infinispan.reactive.publisher.impl.LocalPublisherManagerImpl<K,V>
```